src/views/studio/lib/studio-member-actions.js

```diff
@@ -28,6 +29,7 @@ const normalizeError = (err, body, res) => {
     if (res.statusCode === 403 && body.mute_status) return Errors.USER_MUTED;
     if (res.statusCode === 401 || res.statusCode === 403) return Errors.PERMISSION;
```

Comment:
There are some 403 errors that are not related to the password for this route, and that should result in "general" error messaging, so we may need to add another kind of error here (like Errors.PASSWORD) and find it by checking for "password incorrect" in the message of the body.

Reply:
that makes sense - I was thinking we could add that in a followup

Reply:
sounds good!